merlin/examples/workflows/null_spec/scripts/submit_1_node.sbatch

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+#SBATCH -N 1
+#SBATCH -J MerlinBaseline
+#SBATCH -t 0:10:00
+#SBATCH -p pdebug
+#SBATCH -A lbpm
+#SBATCH --output=merlin%j.out
+#SBATCH --error=merlin%j.err
+#SBATCH --exclusive
+#SBATCH --mpibind=off
+
+SCRIPTPATH=`pwd`
+YAML_FILE=${SCRIPTPATH}/../../../../../null_spec.yaml
+
+# print message that a new run is starting
+echo "Starting new run: $SLURM_JOBID"
+date
+
+# run program
+time merlin run ${YAML_FILE} --vars N_SAMPLES=$1 CONC=$2
+merlin run-workers ${YAML_FILE} --vars N_SAMPLES=$1 CONC=$2
+
+merlin monitor
```

Comment:
For ``merlin monitor`` to work correctly you need an exit ${MERLIN_STOP_WORKERS) task. Without that task, it will just block until the end of the allocation.

Reply:
Thanks for the reminder, will add this.